tic-tac-toe.js

```diff
@@ -1,7 +1,63 @@
 function TicTacToe() {
-
+  this.currentPlayer = "Toad";
+  this.board = [["r1c1", "r1c2", "r1c3"], ["r2c1", "r2c2", "r2c3"], ["r3c1", "r3c2", "r3c3"]];
+  this.play();
 }
 
 TicTacToe.prototype = {
-  
-}
+  play: function() {
+    this.addMarker();
+  },
+
+  addMarker: function() {
+    var self = this;
+    self.board.forEach(function(row, rowIndex) {
+      row.forEach(function(cell, cellIndex) {
+        $("#" + cell).one ("click", function(){
+          $("#" + cell).html('<img src='+self.getMarker()+'>');
+        self.board[rowIndex][cellIndex] = self.getMarker();
```

Comment:
Watch out for your indentation here, i think lines 18-22 are within the anonymous function which is opened on line 16, but it is a bit hard to read that here.